frontend/sw360-portlet/src/main/webapp/html/components/editRelease.jsp

```diff
@@ -244,6 +246,18 @@
 
         $('#formSubmit').click(
             function() {
+                var valid = true;
+                $.each($('#externalIdsTable tr.bodyRow'), function (i, row) {
+                    var key = $('#'+row.id+' td input.keyClass').val();
+                    var value = $('#'+row.id+' td input.valueClass').val();
+                    var isOneEmpty = ((key !== '') && (value === '')) || ((key === '') && (value !== ''));
+                    if (isOneEmpty) {
+                        $.alert('There is a blank field in one of your external ids. Please correct this before updating the release.');
+                        valid = false;
+                        return false;
```

Comment:
I think this return is unnecessary.

Reply:
It prevents multiple alert windows appearing if there's more than one blank field

Reply:
Ah, ok. I have overlooked the loop here.